packages/kbn-es-query/src/filters/helpers/compare_filters.ts

```diff
@@ -45,12 +45,15 @@ const mapFilter = (
     ? removeRequiredAttributes(excludedAttributes)
     : excludedAttributes;
 
-  const cleaned: FilterMeta = omit(filter, attrsToExclude) as FilterMeta;
-
-  if (comparators.index) cleaned.index = filter.meta?.index;
-  if (comparators.negate) cleaned.negate = filter.meta && Boolean(filter.meta.negate);
-  if (comparators.disabled) cleaned.disabled = filter.meta && Boolean(filter.meta.disabled);
-  if (comparators.alias) cleaned.alias = filter.meta?.alias;
+  // Some filters are coming from GET params,
```

Comment:
is this related to other changes in this commit ?

Reply:
Yes. These filters are without undefined values.
<img width="572" alt="Screenshot 2022-12-15 at 16 29 49" src="https://user-images.githubusercontent.com/22456368/207886680-48c66c31-8437-4143-95a6-2e506131bd97.png">